README.md

```diff
@@ -72,6 +135,23 @@ Backends come in two flavors:
   saturate the NIC's bandwidth. Or using a CUDA memcpy call to transfer memory
   from one GPU to another using NVLink.
 
+These different usage patterns promote different design choices when
+implementing transports and channels, which means the two are not perfectly
+interchangeable. For example, a TCP-based transport is best implemented using a
```

Comment:
Shm-based? :)

Reply:
No, I did mean TCP, it's a discussion about the tradeoffs of using one vs more than one sockets for the two use cases.